packages/core/src/stateUtils.ts

```diff
@@ -68,10 +70,6 @@ export function getConfiguration<TC, TE extends EventObject>(
     } else {
       if (s.type === 'parallel') {
         for (const child of getChildren(s)) {
-          if (child.type === 'history') {
```

Comment:
It's now already filtered out by `getChildren` so this could be safely removed from here.